pkg/apis/experimental/types.go

```diff
@@ -639,3 +639,36 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// Selector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty Selector matches all objects. A
+// null Selector matches no objects.
+type Selector struct {
+	// MatchLabels is a list of labels.
+	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+	// MatchExpressions is a list of requirement selectors.
```

Comment:
Two ways you could do OR:
(1) add "Or bool" which means "OR together all of the MatchLabels and MatchExpressions rather than ANDing them" (though as I mentioned in the other comment I don't understand the purpose of MatchLabels)
(2) add "Not bool" which means "after ANDing the MatchExpressions together, invert the result"; this would allow you to express OR of the MatchExpressions (assuming you invert each of the MatchExpressions also) (DeMorgan's Law, the one useful thing I learned in my college digital logic class).